### PR TITLE
[FIX] pos_pricelist: right operator in default pricelist

### PR DESCRIPTION
**Pull request by chienandalu**
Solves this issue: https://github.com/OCA/pos/issues/279

cc @Tecnativa

### CODE REVIEW

File: pos_pricelist/views/pos_config_view.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
                             widget="many2many_tags"/>
                         <field
                             name="pricelist_id"
-                            domain="[('id', 'in', available_pricelist_ids)]"
+                            domain="[('id', 'in', available_pricelist_ids and available_pricelist_ids[0] and available_pricelist_ids[0][2] or False)]"
```
Comment (legalsylvain) [inline pos_pricelist/views/pos_config_view.xml:28]: This domain will return only one id ? Or did  I miss something ? I think it should return all the ids available in the field available_pricelist_ids.

Comment (pedrobaeza) [reply pos_pricelist/views/pos_config_view.xml:28]: No, it returns all the existing ones, as it's stored as [(6, 0, [`<ids>`])], so [0][2] accesses [`<ids>`], but I didn't know this works on the JS part. @chienandalu is it working?

Comment (chienandalu) [reply pos_pricelist/views/pos_config_view.xml:28]: Yes, it is :slightly_smiling_face: 
image

Comment (chienandalu) [reply pos_pricelist/views/pos_config_view.xml:28]: In v11 the right operator is more transparent, anyway (this bug was due to the literal copy of the view in this backport)

Comment (pedrobaeza) [reply pos_pricelist/views/pos_config_view.xml:28]: Thanks for the explanation

**Review by pedrobaeza — APPROVED** [review]
Please change as always module version number

**Comment by chienandalu** [comment]
@pedrobaeza done

**Review by legalsylvain — APPROVED** [review]
@pedrobaeza : thanks for the explanation, I didn't know.
@chienandalu : thanks for this patch !

**Comment by pedrobaeza** [comment]
@legalsylvain as explained by my colleague, on v11 these values were homogenized in the new JS views for x2many widget for being plain lists.

**Pull request merged**